python/MSSMvsSM.py

```diff
@@ -297,6 +305,17 @@ def doHistFuncForGGH(self, varlist):
                 br_htautau = getattr(self.mssm_inputs, self.quantity_map['br']['method'])(accesskey_br, x, y)
                 br_htautau_SM = getattr(self.mssm_inputs, self.quantity_map['br_SM']['method'])(accesskey_br_SM, x, y)
 
+                if not xs_ggh_SM:
```

Comment:
Also here, what does this actually check for?

Reply:
If values are zero ---> not properly filled. This is usual python logic (instead of using `br_htautau != 0`, which is more C++ style.)

Reply:
See the printouts following the clause to understand what is happening.

Reply:
changed... hope it is now better